avogadro/rendering/geometryvisitor.h

```diff
@@ -25,6 +25,12 @@ namespace Rendering {
  * notably the center and radius of the bounding sphere.
  */
 
+#ifdef _3DCONNEXION
```

Comment:
Why are these needed here?

Reply:
They were added to cache scene geometry for hit testing and it's bounding box calculation.

Reply:
It seems to be an another leftover. Removed.